### ISSUE TITLE
Needs to support python 3.11 for komodo on rhel8

### ISSUE THREAD

**Issue by eivindjahren** [equinor/fmu-sumo-sim2sumo#36]
We are starting to build komodo with python 3.11, but get the following problems:
```
fmu-sumo-sim2sumo 0.1.9.dev6+g2d21f34 has requirement opm<=2022.4,>=2020.10.2, but you have opm 2023.10.
fmu-sumo-sim2sumo 0.1.9.dev6+g2d21f34 has requirement pandas<2.0, but you have pandas 2.2.1.
```

**Comment by equinor-ruaj** [comment]
#46 has been merged and should solve the issues with `opm` and `pandas` @eivindjahren

**Comment by eivindjahren** [comment]
Nice is there a tag for which supports python 3.11?

**Comment by equinor-ruaj** [comment]
Released tag [0.1.9](https://github.com/equinor/fmu-sumo-sim2sumo/releases/tag/0.1.9)
